### ISSUE TITLE
RadioConfig Buttons Crash GUI w/ Processing 4

### ISSUE THREAD

**Issue by retiutut** [OpenBCI/OpenBCI_GUI#674]
## Problem

The GUI crashes and a log file is saved in the application directory on Windows.

## Expected 

RadioConfig buttons for Manual Cyton Config need to work on Windows, without the GUI crashing.

## Operating System and Version
Windows

## GUI Version

4.2.0

## Running standalone app

Happens in Standalone and Processing-Java on Windows only

## Type of OpenBCI Board

Cyton and Cyton+Daisy

## Are you using a WiFi Shield?
No

## Console Log
```
Console Log Started at Local Time: 2020-01-31_18-48-18
Screen Resolution: 1920 X 1080
Welcome to the Processing-based OpenBCI GUI!
For more information, please visit: https://openbci.github.io/Documentation/docs/06Software/01-OpenBCISoftware/GUIDocs
Launching application from local data dir
OpenBCI_GUI: hubStart: OS Detected: Windows
TopNav: Internet Connection Successful
Local Version: 420, Latest Version: 420
GUI is up to date!
OpenBCI_GUI::Setup: Sample Data exists in Documents folder.
ControlP5 2.2.6 infos, comments, questions at http://www.sojamo.de/libraries/controlP5
CP: Playback History Size = 1
OpenBCI_GUI::Setup: Setup is complete!
Instantiating hub object...
Hub: CheckHubInit: The GUI is connected to the Hub!
sourceList: click! 
setInterface: curInterface: -1
channel count set to 8
Setting Latency to 10000
Setting WiFi Internet Protocol to udpBurst
Setting WiFi style to dynamic
[DEFAULT]: Protocol Serial Selected for Cyton
setInterface: curInterface: 0
Setting sample rate for Cyton to 250Hz


sample rate set to: 250
[DEFAULT]: Transfer Protocol set to serial
serialList: click! 
[DEFAULT]: OpenBCI Port Name = COM4
Radios_Config: scan_channels
[DEFAULT]: Attempting to open Serial/COM port: COM4
Radios_Config: connect_to_portName: Attempting to open serial port: COM4
```
[hs_err_pid15512.log](https://github.com/OpenBCI/OpenBCI_GUI/files/4141971/hs_err_pid15512.log)

**Comment by retiutut** [comment]
If you need to use this feature on Windows, please use GUI 4.1.7 for now. This should be resolved for GUI 5.0.0.

**Comment by retiutut** [comment]
The upstream issue has been closed! Testing this soon using Processing 4 on Windows.
